challenges/objects-arrays.js

```diff
@@ -59,11 +88,19 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for(let i = 0; i < graduates.length; i++){
+  contactInfo.push(graduates[i].first_name + ' ' + graduates[i].email)
```

Comment:
this works but dont be scared of that shiny template literal syntax ``contactInfo.push(`${graduates[i].first_name} ${graduates[i].email}`)``